scripts/gen_quick_start_module.py

```diff
@@ -163,6 +165,25 @@ def gen_install_matrix(versions) -> Dict[str, str]:
                     result[key] = "<br />".join(lines)
     return result
 
+# This method is used for extracting two latest verisons of cuda and
+# last verion of rocm. It will modify the acc_arch_ver_map object used
+# to update getting started page.
+def extract_arch_ver_map(release_matrix):
+    for chan in ("nightly", "release"):
+        cuda_ver_list = {
+            x["desired_cuda"]: x["gpu_arch_version"] for x in release_matrix[chan]["linux"]
+            if x["gpu_arch_type"] == "cuda"
+        }
+        rocm_ver_list = {
+            x["desired_cuda"]: x["gpu_arch_version"] for x in release_matrix[chan]["linux"]
+            if x["gpu_arch_type"] == "rocm"
+        }
```

Comment:
Avoids code duplication
```suggestion
        def gen_ver_list(gpu_arch_type):
            return {
                x["desired_cuda"]: x["gpu_arch_version"] for x in release_matrix[chan]["linux"]
                 if x["gpu_arch_type"] == gpu_arch_type
             }
        cuda_ver_list = gen_ver_list("cuda")
        rocm_ver_list = gen_ver_list("rocm")
```